math/gmtl/Portfile

```diff
@@ -1,27 +1,66 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem          1.0
 
 name                gmtl
 version             0.6.1
+revision            1
+checksums           rmd160  19ca8c25cf54b1650272f7a387f5f8b8c5c3140c \
+                    sha256  f7d8e6958d96a326cb732a9d3692a3ff3fd7df240eb1d0921a7c5c77e37fc434 \
+                    size    339777
+
 categories          math
-platforms           darwin
-maintainers         iastate.edu:mccdo
+platforms           any
+supported_archs     noarch
+maintainers         {iastate.edu:mccdo @mccdo}
+license             LGPL-2.1+ Restrictive/Distributable
```

Comment:
@jmroot Is this the correct license line? This software claims to be under LGPL-2.1+ but also includes a license addendum that imposes additional restrictions.

Reply:
Most of the license addendum is actually granting additional permissions, which is fine, but section 4 definitely appears to be an additional restriction, which is disallowed by the LGPL. I think this has to be treated as a distinct GPL-incompatible copyleft license, which would be written `Copyleft GPLConflict`.